addons/web/static/src/js/tools/test_menus.js

```diff
@@ -123,13 +123,15 @@
         console.log("Testing menu", element.innerText.trim(), " ", element.dataset.menuXmlid);
         testedMenus.push(element.dataset.menuXmlid);
         var startActionCount = clientActionCount;
-        element.click();
+        _click($(element));
         var isModal = false;
         return waitForCondition(function() {
             // sometimes, the app is just a modal that needs to be closed
             var $modal = $('.modal[role="dialog"][open="open"]');
             if ($modal.length > 0) {
-                $modal.modal('hide');
+                var $close_button = $('header > button.close');
```

Comment:
I would do something like `$modal.find('header > button.close')` to be sure that your close button is in the modal.

Reply:
variables are camelcased in JS